package login.proto;

import "ResultCodes.proto";
import "ClusterInfo.proto";

message GetClusterListReply {
 required ResultCode result_code = 1;
 repeated ClusterInfo cluster_list = 2;
}

